scripts/analytics/preprint_summary.py

```diff
@@ -0,0 +1,69 @@
+import pytz
+
+import logging
+from dateutil.parser import parse
+from datetime import datetime, timedelta
+
+from django.db.models import Q
+from website.app import init_app
+from scripts.analytics.base import SummaryAnalytics
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+LOG_THRESHOLD = 11
+
+
+class PreprintSummary(SummaryAnalytics):
+
+    @property
+    def collection_name(self):
+        return 'preprint_summary'
+
+    def get_events(self, date):
+        super(PreprintSummary, self).get_events(date)
+        from osf.models import PreprintService, PreprintProvider
+
+        # Convert to a datetime at midnight for queries and the timestamp
+        timestamp_datetime = datetime(date.year, date.month, date.day).replace(tzinfo=pytz.UTC)
+        query_datetime = timestamp_datetime + timedelta(1)
+
+        counts = []
+        for preprint_provider in PreprintProvider.objects.all():
```

Comment:
I didn't know the canonical preprint information was kept in SHARE, it makes sense if Preprints aren't going to be dependent on a node. I'll look into it.